nesspoll/src/components/LoginPage.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Redirect } from 'react-router-dom';
+import { TextField, RaisedButton } from 'material-ui';
+
+interface LoginState {
+    username: String;
```

Comment:
type String is reffering to instance of String object. You want to refer to primitive string value. Use lowercase "string"

Reply:
good to know that there is a primitive string :) java does not have that